core/frontend/src/components/parameter-editor/ParameterEditor.vue

```diff
@@ -309,10 +311,29 @@ export default Vue.extend({
         return 'None'
       }
 
-      if (param.options && param.value in param.options) {
-        // TODO: fix this so it doesnt show text for values such as 2.5 (rounding down to 2)
-        return param.options[param.value]
+      // Bitmask can have options for default values, so this needs to go first
+      // E.g: Option: 830 = default, bitmask = ATTITUDE_MED, GPS, PM, CTUN..
+      // TODO: fix this so it doesnt show text for values such as 2.5 (rounding down to 2)
+      const option_value = param.options?.[param.value]
+      if (option_value) {
+        return option_value
       }
+
+      if (param.bitmask) {
+        const bitmask_result = []
+        // We check up to 64 bits to make sure that we are going to cover all possible bits
+        // Including the ones not listed in the bitmask
+        for (let v = 0; v < 64; v += 1) {
+          // eslint-disable-next-line no-bitwise
+          const bitmask_value = 1 << v
+          // eslint-disable-next-line no-bitwise
+          if (param.value & bitmask_value) {
+            bitmask_result.push(param.bitmask?.[v] ?? `Bit ${v + 1}`)
```

Comment:
This could be ok as is, but if someone doesn't know what a bitmask is (just that the parameter has multiple options) it may be worth being more explicit that the value is unknown, not just what it is. Maybe something like
- `UNKNOWN (bit #)`
- `UNKNOWN(#)`
- `Unknown:#`
- `? (bit #)`

Reply:
Done.